apps/schematic/api/pyproject.toml

```diff
@@ -7,21 +7,22 @@ readme = "README.md"
 packages = [{include = "schematic_api"}]
 
 [tool.poetry.dependencies]
-python = "3.11.0"
+python = ">=3.9.0,<3.11"
 connexion = {version = "2.14.1", extras = ["swagger-ui"]}
 swagger-ui-bundle = "0.0.9"
 python-dateutil = "2.8.2"
-Flask = "2.2.2"
-Flask-Cors = "3.0.10"
+Flask = "^1.1.4"
```

Comment:
Pin version number for improved reproducibility?

Reply:
Done.